doc/classes/Button.xml

```diff
@@ -47,7 +47,7 @@
 			If set to something other than [constant TextServer.AUTOWRAP_OFF], the text gets wrapped inside the node's bounding rectangle.
 		</member>
 		<member name="clip_text" type="bool" setter="set_clip_text" getter="get_clip_text" default="false">
-			When this property is enabled, text that is too large to fit the button is clipped, when disabled the Button will always be wide enough to hold the text.
+			When this property is enabled, text that is too large to fit the button is clipped, when disabled the Button will always be wide enough to hold the text. The text is clipped only horizontally, so Button's height is not affected.
```

Comment:
```suggestion
			When enabled, text that is too large to fit the button is clipped horizontally. When disabled, the button will always be wide enough to hold the text. The text is not vertically clipped, and the button's height is not affected by this property.
```
Or more concise:
```suggestion
			When enabled, text that is too large to fit the button is clipped horizontally. When disabled, the button will always be wide enough to hold the text. The text is not vertically clipped, and height is not affected.
```

Or use "If `true`" style, which is recommended generally but this class doesn't currently use (are there exceptions for properties which are usually set in the inspector?):
```suggestion
			If [code]true[/code], text that is too large to fit the button is clipped horizontally. If [code]false[/code], the button will always be wide enough to hold the text. The text is not vertically clipped, and height is not affected.
```

Reply:
There's little exceptions. Perhaps "enabled" reads better in the Project, Editor, and Import settings. But there's no rule about it. The only reason these use "enabled" is because they're old.

Reply:
From a cursory search, ProjectSettings and EditorSettings actually have among the largest counts of "if `true`", likely just from their large size. Regardless, there does not seem to be an exception for those classes.